### DISCUSSION TITLE
Flashfloppy Update and OSD

### DISCUSSION THREAD

**Discussion by meru-zz** [keirf/FF_OSD#54] [Q&A] [answered]
How do i update my Gotek with Flasfloppy if i have this OSD Mod installed?
For an update I have to press the 2 buttons on the Gotek, but the Gotek is no longer accessible from the outside.
Is there a possibility to initiate the update of the firmware via the keyboard or the bluepill?

**Comment by keirf** [comment, marked as answer]
Press select hotkey when no usb stick is inserted. Brings up a main menu from which you can select Update Firmware. Click a few times, insert USB stick with UPD file, and away you go.

**Reply by meru-zz** [reply]
ok, i will try it 
thx
